paging/paging-common/src/test/kotlin/androidx/paging/PagingDataDifferTest.kt

```diff
@@ -2185,28 +2175,25 @@ class PagingDataDifferTest(
     }
 
     private fun runTest(
-        loadDispatcher: TestDispatcher = StandardTestDispatcher(),
         initialKey: Int? = null,
-        pagingSources: MutableList<TestPagingSource> = mutableListOf(),
-        pager: Pager<Int, Int> =
-            Pager(
-                config = PagingConfig(pageSize = 3, enablePlaceholders = false),
-                initialKey = initialKey,
-                pagingSourceFactory = {
-                    TestPagingSource(
-                        loadDelay = 1000,
-                        loadContext = loadDispatcher,
-                    ).also { pagingSources.add(it) }
-                }
-            ),
-        block: (
+        block: TestScope.(
             differ: SimpleDiffer,
-            loadDispatcher: TestDispatcher,
             pagingSources: List<TestPagingSource>,
             uiReceivers: List<TrackableUiReceiverWrapper>,
             hintReceivers: List<TrackableHintReceiverWrapper>
         ) -> Unit
     ) = testScope.runTest {
+        val pagingSources = mutableListOf<TestPagingSource>()
+        val pager = Pager(
+            config = PagingConfig(pageSize = 3, enablePlaceholders = false),
+            initialKey = initialKey,
+            pagingSourceFactory = {
+                TestPagingSource(
+                    loadDelay = 1000,
+                    loadContext = coroutineContext,
```

Comment:
I think we no longer need to set `loadContext` here

Reply:
Ooo nice, just removed it in the latest commit!